### ISSUE TITLE
Unbind computed id from start_url

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/manifest#1148]
It would be good to unbind the computed id from the start_url, as the start_url could change. This leads to an issue where one set of users end up with one id, and a future set of users end up with a different id.

**Comment by benfrancis** [comment]
@marcoscaceres I agree this is a problem. However, considering the years-long [analysis](https://github.com/philloooo/pwa-unique-id/blob/main/explainer.md) and [discussion](https://github.com/w3c/manifest/issues/586) which arrived at the current solution, do you have an alternative in mind?

The biggest obstacle to changing it seems to be backwards compatibility with existing implementations.

A future Web App Manifest 2.0 specification could potentially make a breaking change. E.g. Make manifest URL the default identifier of a web app (my preference, requiring other significant simplifications), or make the `id` member mandatory (probably a more likely consensus).

**Comment by dmurph** [comment]
For reference:  https://bit.ly/installed-web-app-state-struggles-2024-presentation

**Comment by dmurph** [comment]
I don't think this is possible if we want to remain compatible with the years of installable pwas that are already out there, sadly.

**Comment by dmurph** [comment]
Also, for reference, we had a huge discussion about this already, explainer here: https://github.com/philloooo/pwa-unique-id/blob/main/explainer.md

The world is, and was, full of PWAs that change their manifest url every version, and also people use data urls for manifests. So we were stuck with this solution from the very beginning.

So I don' think this is revisit-able.
